Features/GoogleAnalytics/GoogleAnalytics.php

```diff
@@ -18,37 +25,23 @@ public function __construct($options)
         if ($this->gaId && $this->isValidId($this->gaId)) {
             add_action('wp_footer', [$this, 'addScript'], 20, 1);
         } else if ($this->gaId != 1 && !isset($_POST['acf'])) {
-            trigger_error('Invalid Google Analytics Id: ' . $this->gaId, E_USER_WARNING);
+            trigger_error("Invalid Google Analytics Id: {$this->gaId}", E_USER_WARNING);
         }
```

Comment:
Strange, then this was already broken because I didn't change it. Thanks for the fix!